workloads/network-perf-v2/env.sh

```diff
@@ -1,11 +1,17 @@
 # Common
 export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
 export UUID=${UUID:-$(uuidgen)}
+export CLEAN_UP=${CLEAN_UP:-true}
+export LOCAL=${LOCAL:-false}
 
 # k8s-netperf version
+export ALL_SCENARIOS=${ALL_SCENARIOS:-true}
+export DEBUG=${DEBUG:-true}
+export METRICS=${METRICS:-true}
+export NETPERF_FILENAME=${NETPERF_FILENAME:-k8s-netperf}
 export NETPERF_VERSION=${VERSION:-v0.1.18}
 export OS=${OS:-Linux}
-export ARCH=$(arch)
+export ARCH=$(uname -m)
```

Comment:
not sure the container images in Prow have `uname` -- I think we used to have `uname` but ran into issues in Prow? I might be misremembering though.

Reply:
Pretty sure they're both in `coreutils-single` in RHEL. `arch` is more RHEL specific where `uname` is everywhere.

Reply:
😿 I think they don't use RHEL container images... 

Looking at the output from one of the runs in prow.
```
+ cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
+ oc config view
```

Reply:
Not using RHEL container images, yikes.

```
$ podman run -it debian:11
root@21182cf1c235:/# uname -m
x86_64
root@21182cf1c235:/# which uname
/bin/uname
```